sagemaker_model_monitor/fairness_and_explainability_jsonlines/SageMaker-Monitoring-Bias-Drift-for-Endpoint.ipynb

```diff
@@ -0,0 +1,1949 @@
+{
```

Comment:
<ol><li>Can we make <code>https://jmespath.org</code> a hyperlink instead?</li><li>How about keeping <code>data.features.values</code> and <code>data.label</code> as inline code?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>Thanks. </p><p> </p><p>For the first one, I wanted to add both inline code mark and hyperlink mark but unfortunately they don't work together (https://github.com/spatialaudio/nbsphinx/issues/301). I still want to keep the inline code mark, so I will change it to this,</p><p> </p><p>* Instead, it is a <code>JMESPath</code> expression ([refer to its specification](https://jmespath.org/specification.html)) that is used to locate the features list or the ground truth label in the dataset.</p><p> </p><p>For the second one, as you suggested I will enclose the expression strings by inline code mark,</p><p> </p><p>* then the <code>features</code> parameter should uses value <code>"data.features.values"</code>, and the <code>label</code> parameter should use value <code>"data.label"</code>. </p><p> </p><p> </p><p> </p><p> </p>